userspace/libsinsp/filter/ast.cpp

```diff
@@ -16,9 +16,143 @@ limitations under the License.
 */
 
 #include "ast.h"
+#include "sinsp_exception.h"
 
 using namespace libsinsp::filter::ast;
 
+std::string libsinsp::filter::ast::unescape_str(const std::string& str)
```

Comment:
As for my other comment, I think both `unescape_str()` and `escape_str` should be defined in `parser.cpp`. This incidentally would also save the git history for `unescape_str()` too.